src/licensedcode/data/licenses/linux-man-pages-copyleft-var.LICENSE

```diff
@@ -0,0 +1,27 @@
+---
+key: linux-man-pages-copyleft-var
+short_name: Linux man-pages Variant
+name: Linux man-pages Copyleft Variant
+category: Copyleft
+owner: Free Software Foundation (FSF)
+spdx_license_key: Linux-man-pages-copyleft-var
+other_urls:
+    - https://git.kernel.org/pub/scm/docs/man-pages/man-pages.git/tree/man2/set_mempolicy.2#n5
+---
+
```

Comment:
This is very similar also to [`verbatim-manual`](https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/verbatim-manual.LICENSE) license and replaces the replaces `verbatim-manual_7.RULE`,
in which the third para contains the following extra sentence:

```
The author(s) may not
have taken the same level of care in the production of this manual,
which is licensed free of charge, as they might when working
professionally.
```
It also says: `notes: the latext2e, verbatim-manual and abstyles are similar licenses`

Also not sure about the license key, for this license, and also the previous 2 licenses:
- `linux-man-pages-2-para`
- `linux-man-pages-1-para`

Reply:
Yes, these seem odd, and perhaps excessively detailed, but we should go ahead and accept them.